### DISCUSSION TITLE
How to integrate Unocss into Quasar CLI with Vite flavor?

### DISCUSSION THREAD

**Discussion by murugappanrm** [unocss/unocss#1482] [Q&A] [answered]
Hi

I am a newbie to UNOCSS and very excited with its capabilities. I am trying integrate UNOCSS into Quasar CLI with Vite flavor (not the webpack option). I am totally lost in this as the guide in UNOCSS mention vite installation but this will not work with QuasarCLI with Vite

Has anyone done this and appreciate any help if you have?

**Comment by SnowingFox** [comment]
https://github.com/unocss/unocss/tree/main/examples/vite-vue3

**Reply by murugappanrm** [reply]
@SnowingFox ,

Thank you for answer but that is what already know. The example is not for Quasar.

**Comment by murugappanrm** [comment, marked as answer]
Look like there is not much interest in Quasar or not many user of Quasar are interested in new addons/plugins. Please dont take me wrong. I am making an assumption based on the frequency of answered issues.

**Reply by philippedasilva-orizone** [reply]
Hey @murugappanrm, well there is interest at least from my side 😛 
A quick search just on github and its issues filter would have shown a few people trying to solve it.
I managed to deal with it when I got introduced to this issue a while back :
https://github.com/unocss/unocss/issues/790

It should bring you onto the right direction ;)

**Reply by murugappanrm** [reply]
Hi @philippedasilva-orizone ,

Thank you so much. Yeah this issue was resolved under #790.  I found this the same way you did and it worked. Sadly, it was negligence on my part for not updating the status and got @SnowingFox hooked in. So sorry folks.
